lib/net/http_fetch_plugin.js

```diff
@@ -153,7 +156,7 @@ shaka.net.HttpFetchPlugin = class {
           // progressUpdated().
           if (currentTime - lastTime > 100 || readObj.done) {
             progressUpdated(currentTime - lastTime, loaded - lastLoaded,
-                contentLength - loaded);
+                contentLength - loaded, contentLength);
```

Comment:
In this case, bytesRemaining = contentLength - loaded, newSegmentSize = contentLength, so bytesRemaining > newSegmentSize will always be false. Am I understanding something wrong? (StreamingEngine.js Line 708)

Reply:
hmm look like you are right
this will switch only on safeBufferedAhead check